File: INICIANTE/1013/main.js
function maior(a, b) {
  return (a + b + Math.abs(a - b)) / 2;
}

let [a, b, c] = require('fs').readFileSync('/dev/stdin', 'utf-8').split(' ').map(Number);
console.log(`${maior(maior(a, b), c)} eh o maior`);

File: INICIANTE/1013/Main.java
import java.util.Scanner;

public class Main {

    static int maior(int a, int b) {
        return (a + b + Math.abs(a - b)) / 2;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int c = scanner.nextInt();
        int greatest = maior(maior(a, b), c);
        System.out.printf("%d eh o maior\n", greatest);
        scanner.close();
    }
}